libr/core/cconfig.c

```diff
@@ -3261,6 +3261,9 @@ R_API int r_core_config_init(RCore *core) {
 	SETCB ("dir.home", p, &cb_dirhome, "Path for the home directory");
 	free (p);
 	p = r_sys_getenv (R_SYS_TMP);
+	if (!p) {
```

Comment:
just do as done in other places:
```C
SETCB ("dir.tmp", p? p: "", &cb_dirtmp, "Path of the tmp directory");
```

Reply:
yeah thats simpler, lets see if travis is happy and ill commit this